Changes.md

```diff
@@ -5,6 +5,7 @@ API
 ---
 
 - PlugLayout : Added a warning for plugs that reference activators that have not been registered.
+- ImageTestCase : `assertImagesEqual()` now uses an order-independent check when asserting that `channelNames` are the same.
```

Comment:
Hmmm, I wonder if this should be optional?  It's reasonable that round tripping to file doesn't preserve order of channel names, but there are lots of things in Gaffer that should produce a consistent order of channel names - I'm thinking about something like merging images with mismatched channels ... I guess we could test the channelNames separately if it matters, but I feel like I can vaguely imagine an improbable bug that could slip through due to this change.  Maybe not worth worrying about, but there would be the option of having a parameter to assertImagesEqual.

Reply:
Fair point - squashed a parameter into 362e52c.